schema/bom-1.7.xsd

```diff
@@ -554,12 +554,25 @@ limitations under the License.
                         of the component. Examples: commons-lang3 and jquery</xs:documentation>
                 </xs:annotation>
             </xs:element>
-            <xs:element name="version" type="bom:versionType" minOccurs="0" maxOccurs="1">
-                <xs:annotation>
-                    <xs:documentation>The component version. The version should ideally comply with semantic versioning
-                        but is not enforced.</xs:documentation>
-                </xs:annotation>
-            </xs:element>
+            <xs:choice minOccurs="0" maxOccurs="1">
+                <xs:element name="version" type="bom:versionType">
+                    <xs:annotation>
+                        <xs:documentation>The component version. The version should ideally comply with semantic versioning
+                            but is not enforced.</xs:documentation>
+                    </xs:annotation>
+                </xs:element>
+                <xs:element name="versionRange" type="bom:versionRangeType">
+                    <xs:annotation>
+                        <xs:documentation><![CDATA[
+                            For an extraneous component, this is the accepted version range.
+                            Value must follow Package URL Version Range syntax (vers)which is defined at <https://github.com/package-url/purl-spec/blob/master/VERSION-RANGE-SPEC.rst>.
+                            The component version range that may be provided to fulfill this capability.
+                            May only occur if attribute `@isExtraneous` is set to `true`.
+                            Must not be used for `/metadata/component`.
```

Comment:
@mrutkows pointed out that the wollowing MUST be added: 
Formulations (MBOM) MUST NOT rely on extraneous components.

need to add this, still!

Reply:
fixed the wording in away that makes clear that only runtime components shoulduse this.